source/documentation/using_the_api/register_components.md

```diff
@@ -1,18 +1,34 @@
-## The relationship between entries, items and records
+## The components of a register
 
-Registers are made up of:
+Registers are primarily composed of:
 
 * entries
 * items
 * records
 
+### Entries
+
 Entries contain metadata about what and when data has changed. 
 
-Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
+
+The `entry-number` is unique and defines an entry’s position within the ordered list of a register. The `entry-timestamp` is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The `entry-number` is.
+
+Entries are connected to items by a given `item-hash`. 
+
+Each `key` is an alphanumeric string unique to the data in a register. The primary key (i.e. the unique identifier) always has the same name as the name of the register. For example, every entry in the `local-authority-eng` register has a field called `local-authority-eng`. Foreign keys, which are not always present, link to other registers. 
```

Comment:
I realised we have keys that contain other carachters than just alphanumeric. I need to see how many registers are afected by this. /cc @gidsg @nacnudus